front/src/app/todo-list/todo-list.component.scss

```diff
@@ -161,3 +160,11 @@
     }
   }
 }
+
+.sidebar-button {
+  ::ng-deep {
+    .mat-icon {
+      margin: 0px;
+    }
+  }
+}
```

Comment:
1. Avoid using ng-deep when possible, in this case you should add a class to mat-icon element and style it
2. A file should have an empty line in the very end